Project.toml

```
@@ -1,12 +1,16 @@
 name = "Tau"
 uuid = "c544e3c2-d3e5-5802-ac44-44683f340e4a"
-version = "0.2.0"
+version = "0.3.0"
```

Once we make a breaking release I'd go to v1.0

---

I don't mind :slightly_smiling_face: 
```suggestion
version = "1.0.0"
```